src/libraries/ErrorsLib.sol

```diff
@@ -27,6 +27,9 @@ library ErrorsLib {
     /// @notice Thrown when the market is already created.
     string internal constant MARKET_ALREADY_CREATED = "market already created";
 
+    /// @notice Thrown when a token to transfer doesn't have code.
+    string internal constant NO_CODE = "no code";
```

Comment:
Nitpick: the error lib is a library shared between different contracts, consider using a more specific error name (and error message). In this case, it could be `CONTRACT_NOT_EXISTING` or `CONTRACT_NO_CODE`

Reply:
nitpicking goes brrrr